Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css

```
@@ -1,319 +1,119 @@
-/* Style applied to the TabbedMenu. */
+.tabbedMenu
+{
+  --tabbedMenu-seperator: var(--remotion-themed-border-width) solid var(--color-contrast-separator);
+  --tabbedMenu-min-height: calc(var(--remotion-themed-spacing) * 7);
+}
+
 table.tabbedMenu
 {
   width: 100%;
   border-collapse: collapse;
   border-spacing: 0;
 }
 
-td.tabbedMainMenuCell
+table.tabbedMenu tr
 {
-  padding: 0;
-  padding-top: 0.25em;
-  background-color: White;
+  border-top: var(--tabbedMenu-seperator);
+  border-bottom: var(--tabbedMenu-seperator);
 }
 
-td.tabbedSubMenuCell
+td.tabbedMainMenuCell
 {
-  background-color: White;
-  padding: 0.3em;
-  border-bottom: solid 1px #AAAAAA;
+  background-color: var(--color-contrast);
 }
 
-td.tabbedSubMenuCell ul
+td.tabbedMainMenuCell,
+td.tabbedSubMenuCell
 {
-  overflow: hidden;
+  padding: var(--remotion-themed-spacing) var(--remotion-themed-spacing-double) 0;
```

I think this was originally based on this comment here: https://github.com/re-motion/Framework/pull/199#discussion_r708291953

I was going with the following to keep things readable to the non-intellisense aware observer:
```suggestion
  padding-top: var(--remotion-themed-spacing);
  padding-bottom: 0;
  padding-left: var(--remotion-themed-spacing-double);
  padding-right: var(--remotion-themed-spacing-double);
```